cli_tools/common/utils/storage/scratch_bucket_creator.go

```diff
@@ -26,8 +26,9 @@ import (
 )
 
 const (
-	defaultRegion       = "US"
-	defaultStorageClass = "MULTI_REGIONAL"
```

Comment:
The way this const is now used, renaming it to multiregionalStorageClass makes more sense

Reply:
They are still default. We are still using defaultRegion and defaultStorageClass when we can't determine region by any input.